webapp/packages/core-blocks/src/useObservableRef.ts

```diff
@@ -70,12 +70,12 @@ export function useObservableRef<T extends Record<any, any>>(
       update = undefined;
     }
 
-    state = makeObservable(state, observed, { deep: false, name });
-
     if (bind) {
       bindFunctions(state, bind as []);
     }
 
+    state = makeObservable(state, observed, { deep: false, name });
```

Comment:
this one moved cause makeObservable freeze methods (`Object.freeze`) in the `state`
so this makes jest falls in `bindFunction` due to the fact we literally reasign methods with its binded version

Reply:
it's incorrect because now if you will test it on update (`rerender`) it will fail and we will get different behaviour for init / update